bazel/patches/wasmtime-0.20.0-fix-include-path.diff

```diff
@@ -0,0 +1,28 @@
+diff --git a/crates/c-api/include/wasi.h b/crates/c-api/include/wasi.h
+index 92271496c..35a29069f 100644
+--- a/crates/c-api/include/wasi.h
++++ b/crates/c-api/include/wasi.h
+@@ -7,7 +7,7 @@
+ #ifndef WASI_H
+ #define WASI_H
+ 
+-#include "wasm.h"
++#include "wasmtime/include/wasm.h"
+ 
+ #ifndef WASI_API_EXTERN
+ #ifdef _WIN32
+diff --git a/crates/c-api/include/wasmtime.h b/crates/c-api/include/wasmtime.h
+index 75880ce02..21bb3c6f0 100644
+--- a/crates/c-api/include/wasmtime.h
++++ b/crates/c-api/include/wasmtime.h
+@@ -12,8 +12,8 @@
+ #ifndef WASMTIME_API_H
+ #define WASMTIME_API_H
+ 
+-#include <wasm.h>
+-#include <wasi.h>
++#include "wasmtime/include/wasm.h"
++#include "wasmtime/crates/c-api/include/wasi.h"
```

Comment:
Ditto.

Reply:
fixed (I remove wasmtime specific dependency since I no longer need wat2_wasm)

https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/73/commits/9f267ef0b2c1a71b1b884afa9dc922968933a787
https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/73/commits/e6cea790d7e0fe8e66fd240e486350c16503b695